p2p/src/authenticated/actors/tracker/ingress.rs

```diff
@@ -150,11 +162,16 @@ impl<E: Spawner + Metrics, C: Verifier> Reservation<E, C> {
 impl<E: Spawner + Metrics, C: Verifier> Drop for Reservation<E, C> {
     fn drop(&mut self) {
         let (peer, mut mailbox) = self.closer.take().unwrap();
-        self.context
-            .clone()
```

Comment:
Not only do we not need to clone anymore (with `spawn_ref`) but this clones twice (`with_label` clones implicitly).